docs/ensembl-help/using-ensembl/ensembl-apps/genome-browser/tracks.md

```diff
@@ -13,9 +13,39 @@ status: draft
 
 # Tracks in the Genome browser
 
-Tracks are sets of data that can be plotted against the genome, such as transcripts, genetic variation or conserved regions. Entities within tracks are shown on horizontal rows. There may be multiple rows for a track, ensuring that there is no overlap within a row. Some tracks, such as genes, are associated with one strand of the genome – behind stranded tracks you will see small arrows pointing in the direction of transcription.
+## What is a track ? 
```

Comment:
Remove the space before the question mark.

Reply:
done